lib/scope.js

```diff
@@ -616,12 +622,16 @@ module.exports = {
         values.push( node.attribs.value );
       }
 
+      // TODO: rename `rows` to `guesses`?
       field_data.rows = await scope.getPossibleTableRows( scope, { node, var_names, values });
       page_data.fields.push( field_data );
 
     }  // ends for every node
 
-    // console.log( JSON.stringify( page_data, null, 2 ));
+    if ( page_data.sought_var[0] === `direct_showifs`) {
+      // console.log( field_like_names );
+      // console.log( JSON.stringify( page_data, null, 2 ));
+    }
```

Comment:
This looks like debugging code that isn't doing anything anymore.

Reply:
I should have mentioned this and thanks for spotting unused code. It's not doing anything here anymore, but I continue to use it to help build further tests. I log the objects created, check over them to make sure they look right, then use them to build the `page_data.fixtures.js` values.